src/main/scala/uk/gov/hmrc/mongo/ReactiveRepository.scala

```diff
@@ -35,19 +31,19 @@ abstract class ReactiveRepository[A <: Any, ID <: Any](collectionName: String,
                                                        idFormat: Format[ID] = ReactiveMongoFormats.objectIdFormats,
                                                        mc: Option[JSONCollection] = None)
                                                       (implicit manifest: Manifest[A], mid: Manifest[ID])
-  extends Repository[A, ID] with Indexes {
+  extends Repository[A, ID] with Indexes with IndexUpdate {
 
   import play.api.libs.json.Json.JsValueWrapper
-  import scala.concurrent.ExecutionContext.Implicits.global
   import reactivemongo.core.commands.GetLastError
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
The Enumeration has been removed as no longer needed since the last refactoring.